templates/myid.j2

```diff
@@ -1 +1,5 @@
-{{ zoo_id }}
```

Comment:
This could be a one-liner using a jinja filter but the complexity of that line got out of hand so we went with the easy-to-read alternative.